common/djangoapps/student/migrations/0053_auto__add_field_userprofile_dropdown.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring,line-too-long,unused-argument
```

Comment:
We ignore migrations files when running pylint. If you run `paver run_quality` locally without first running `paver run_pylint`, you may get erroneous reports on migrations files. However, you should leave migration files alone from how they're generated.